Minor N2 atmos piping change on MetaStation

dablusniper opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Slightly changed the piping of the N2 to Airmix and N2 to Pure pipes in atmos to make them look more orderly and consistent with the O2 piping.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Makes atmos look more organised.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
tweak: Slightly changed N2 to Airmix/Pure piping in atmos on Metastation to make it look more organised.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

dablusniper committed:
Small N2 atmos piping change on MetaStation

Slightly changed the piping of the N2 to Airmix and N2 to Pure pipes to make them look more orderly and consistent with the O2.

trollbreeder commented on _maps/map_files/MetaStation/MetaStation.dmm:
```diff
@@ -46736,6 +46724,11 @@
+/obj/machinery/atmospherics/components/binary/pump{
+	dir = 1;
+	name = "N2 to Airmix";
+	on = 1
```
use /obj/machinery/atmospherics/components/binary/pump/on instead, if you dont it'll look like it's turned off in the map editor

dablusniper replied:
Fixed, thanks.

dablusniper committed:
Replaced pumps with existing "on" instance in DM

Changed pumps from previously used new instances to existing "on" pump instances

LemonInTheDark commented:
Someone give this man a good first pr
